src/wfdirsrc.c

```diff
@@ -286,9 +286,9 @@ DSDragLoop(HWND hwndLB, WPARAM wParam, LPDROPSTRUCT lpds)
    hwndGlobalSink = lpds->hwndSink;
 
    //
-   // default to copy
+   // default to move
    //
-   bShowBitmap = TRUE;
+   bShowBitmap = FALSE;

```

Comment:
:-) Ha! This is a super old bug and was there ever since. It took me hours to figure it out:-)

In the previous code bShowBitMap was initialized with TRUE aka copy, but the cursor is loaded for 'Move' due to iCurDrag initially. So they were out of sync.  On the first move of the mouse I assume iCurDrag was then 'copy' (if you pressed CTRL) and they got it in sync with wfdirsrc.c:393 

If you have the original coding in wfdirsrc.c:1048 this will work, but in the new case you have to start with cursor and bShowBitMap  in sync